Dockerfile_arm64

```diff
@@ -0,0 +1,18 @@
+# This is a two-stage Docker build:
+# https://docs.docker.com/develop/develop-images/multistage-build
+
+#
+# Build container
+FROM golang:1.14 AS builder
+
+WORKDIR /src
+COPY . .
+RUN ./linux-build-arm64.sh
```

Comment:
The only difference between linux-build.sh and linux-build-arm.sh is the addition of the GOARCH variable. Would it be better to instead add `ARG GOARCH=amd64` to the Dockerfile, and then override it when you want to build for arm64? Then we would not need two Dockerfiles, and you could instead run:
`docker build --build-arg GOARCH=arm64 --file=Dockerfile .`

Reply:
OK, I've adapted the logic. Tell me what you think! I tested it, it works on the Pi.